src/one_dragon/utils/http_utils.py

```
@@ -48,22 +79,70 @@ def log_download_progress(downloaded_bytes: int, total_size: int) -> None:
         if progress_callback is not None:
             progress_callback(0, msg)
+        if status_callback is not None:
+            status_callback(HttpDownloadProgress(phase='connecting', message=gt('正在连接')))
 
         url = urllib.parse.urlparse(download_url)
         if url.scheme not in ('http', 'https'):
             raise ValueError(f"不支持的下载协议：{download_url}")
 
         request = urllib.request.Request(download_url)
-        with opener.open(request, timeout=60) as response:
+        # 读超时保持较短，避免网络停滞时取消请求长时间无响应。
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)